pandas/__init__.py

```diff
@@ -236,6 +248,28 @@ class SparseDataFrame:
     class SparseSeries:
         pass
 
+    class __numpy:
+        def __init__(self):
+            import numpy as np
+            import warnings
+
+            self.np = np
+            self.warnings = warnings
+
+        def __getattr__(self, item):
+            self.warnings.warn(
```

Comment:
im getting a mypy complaint about this.  why use self.warnings rather than just warnings?